.github/workflows/other-os-build.yml

```diff
@@ -0,0 +1,256 @@
+# OpenMLDB build on other os, only trigger by dispatch event and only upload artifacts, not release.
+# You should download artifacts manually. And we won't do test in this workflow.
+name: Other OS Build
+
+on:
+  workflow_dispatch:
+    inputs:
+      OS:
+        description: 'os name: centos6, macos11, macos10'
+        required: true
+        default: 'centos6'
+      JAVA_SDK:
+        description: 'java sdk enable: ON, OFF'
+        required: true
+        default: 'ON'
+      PYTHON_SDK:
+        description: 'python sdk enable: ON, OFF'
+        required: true
+        default: 'ON'
+
+env:
+  GIT_SUBMODULE_STRATEGY: recursive
+  NPROC: 2 # default Parallel build number for GitHub's Linux runner
+  EXAMPLES_ENABLE: OFF # turn off hybridse's example code
+  HYBRIDSE_TESTING_ENABLE: OFF # turn off hybridse's test code
+
+jobs:
+  centos6: # glibc version is 2.12, glibcxx version is 3.4.13
+    if: ${{ github.event.inputs.OS == 'centos6' }}
+    runs-on: ubuntu-latest
+    env:
+      OS: linux
+      SQL_JAVASDK_ENABLE: ${{ github.event.inputs.JAVA_SDK }}
+      SQL_PYSDK_ENABLE: ${{ github.event.inputs.PYTHON_SDK }} # python whl will be built when make, no prerequirement
+      TESTING_ENABLE: OFF
+      NPROC: 8
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
give the option  to build upon release tag, and tweak release artifact name

Reply:
You mean a option `VERSION` to let user define the release version?

Reply:
done